docs/topics/wildfly-swarm/docs/reference/fractions/elytron.adoc

```diff
@@ -0,0 +1,996 @@
+
+swarm.elytron.simple-role-decoders._KEY_.attribute:: 
+The name of the attribute from the identity to map directly to roles.
+
+swarm.elytron.size-rotating-file-audit-logs._KEY_.attribute-synchronized:: 
+Whether every event should be immediately synchronised to disk.
+
+swarm.elytron.size-rotating-file-audit-logs._KEY_.format:: 
+The format to use to record the audit event.
+
+swarm.elytron.size-rotating-file-audit-logs._KEY_.max-backup-index:: 
+The maximum number of files to backup when rotating.
+
+swarm.elytron.size-rotating-file-audit-logs._KEY_.path:: 
+Path of the file to be written.
+
+swarm.elytron.size-rotating-file-audit-logs._KEY_.relative-to:: 
+The relative path to the audit log.
+
+swarm.elytron.size-rotating-file-audit-logs._KEY_.rotate-on-boot:: 
+Whether the file should be rotated before the a new file is set.
+
+swarm.elytron.size-rotating-file-audit-logs._KEY_.rotate-size:: 
+The log file size the file should rotate at.
+
+swarm.elytron.size-rotating-file-audit-logs._KEY_.suffix:: 
+Format of date used as suffix of log file names in java.time.format.DateTimeFormatter. The suffix does not play a role in determining when the file should be rotated.
+
+swarm.elytron.syslog-audit-logs._KEY_.format:: 
+The format to use to record the audit event.
+
+swarm.elytron.syslog-audit-logs._KEY_.host-name:: 
+The host name to embed withing all events sent to the remote syslog server.
+
+swarm.elytron.syslog-audit-logs._KEY_.port:: 
+The listening port on the syslog server.
+
+swarm.elytron.syslog-audit-logs._KEY_.server-address:: 
+The server address of the syslog server the events should be sent to.
+
+swarm.elytron.syslog-audit-logs._KEY_.ssl-context:: 
+The SSLContext to use to connect to the syslog server when SSL_TCP transport is used.
+
+swarm.elytron.syslog-audit-logs._KEY_.transport:: 
+The transport to use to connect to the syslog server.
+
+swarm.elytron.token-realms._KEY_.jwt:: 
+A token validator to be used in conjunction with a token-based realm that handles security tokens based on the JWT/JWS standard.
+
+swarm.elytron.token-realms._KEY_.oauth2-introspection:: 
+A token validator to be used in conjunction with a token-based realm that handles OAuth2 Access Tokens and validates them using an endpoint compliant with OAuth2 Token Introspection specification(RFC-7662).
+
+swarm.elytron.token-realms._KEY_.principal-claim:: 
+The name of the claim that should be used to obtain the principal's name.
+
+swarm.elytron.trust-managers._KEY_.algorithm:: 
+The name of the algorithm to use to create the underlying TrustManagerFactory.
+
+swarm.elytron.trust-managers._KEY_.alias-filter:: 
+A filter to apply to the aliases returned from the KeyStore, can either be a comma separated list of aliases to return or one of the following formats ALL:-alias1:-alias2, NONE:+alias1:+alias2
```

Comment:
format list should use backticks

Reply:
Note to self: not easily fixable in our sources.